components/card/readme.md

```diff
@@ -58,3 +58,9 @@ Then your style should have:
 
 Please see this example: [Grid Example](https://vivid.vonage.com?path=/story/alpha-components-card--in-a-grid)
 
+## CSS Custom Properties
+
+| Property                       | Default                                          | Description                                      |
+|--------------------------------|--------------------------------------------------|--------------------------------------------------|
+| `--title-line-number` | none | Controls the number of lines of the title that will be present before trim & ellipsis |
+| `--subtitle-line-number` | none | Controls the number of lines of the sub-title that will be present before trim & ellipsis |
```

Comment:
why wont we stick to the css property name here? `line-clamp`

Reply:
because than the same number will be for both title & subtitle